### ISSUE TITLE
WPF - fails to provide visual focus indicators (text caret, text input border) at some sites

### ISSUE THREAD

**Issue by tentakles** [cefsharp/CefSharp#4146]
**Problem description:**

When visiting some sites, An application built with CefSharp WPF does not visually indicate that a text input is focused (no text caret, no focused input border).- 

This can be reproduced in the latest "CefSharp MinimalExample WPF", v102 and earlier versions such as v97.- 

With "CefSharp MinimalExample WinForms" visual indicators are provided correctly, so this problem seems to exist only for the WPF version.

**Steps to reproduce:**

- Open "CefSharp MinimalExample WPF" on a Win10 x64 machine.
- Navigate to this site:
- https://internetbanken.privat.nordea.se/nsp/login
- click the button "Till inloggning"
- click "e-kod"

Result: the code text input does not show visual focus indication such as caret and text input border, however it is possible to write into the text box.

Expected result: the text input should show a text caret and a highlight border, like it does in regular Chrome and also in "CefSharp MinimalExample WinForms"

This also occurs on other non public sites.
	
It appears that workarounds is available: 
Switch focus to another app and back again to make it work, or focus the address bar in CefSharp.MinimalExample and back again to make the focus indicating work again
	
**Environment:**
OS: Windows 10 x64
CefSharp version 102
.NET 4 and core

It occurs on prior version but is reproducible with CefSharp.MinimalExample v102 from github.

**Information from debug log:**
No relevant information found in debug log

**Reproducible in CEF Sample Application:**
No, only for CefSharp WPF

**Comment by amaitland** [comment]
A very quick test and you can potentially manually call `SetFocus(true)` after the page has finished loading as a workaround. Obviously this has had minimal testing and will still need to be fixed in `CEF`.

```c#
Browser.LoadingStateChanged += OnBrowserLoadingStateChanged;

private void OnBrowserLoadingStateChanged(object sender, LoadingStateChangedEventArgs e)
{
	if(!e.IsLoading)
	{
		e.Browser.GetHost().SetFocus(true);
	}
}
```

**Comment by EricPRHC** [comment]
Hi @amaitland, logging into bitbucket won't allow me to see your refered page https://bitbucket.org/chromiumembedded/cef/issues?status=new&status=open.
I'm very curious as to what the status of this issue is.

**Comment by amaitland** [comment]
The CEF Issue Tracker moved to GitHub (#4430)

**Comment by EricPRHC** [comment]
Couldnt find the issue there so I created it
https://github.com/chromiumembedded/cef/issues/3510
